### PR TITLE
Add new model, accton-as5912-54xk.

### PR DESCRIPTION
**Pull request by brandonchuang**

### CODE REVIEW

File: packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/auto/x86_64_accton_as5912_54xk.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+###############################################################################
+#
+# x86_64_accton_as5912_54xk Autogeneration Definitions.
+#
+###############################################################################
+
+cdefs: &cdefs
+- x86_64_accton_as5912_54xk_CONFIG_INCLUDE_LOGGING:
```
Comment (jnealtowns) [inline packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/auto/x86_64_accton_as5912_54xk.yml]: @brandonchuang  Please make the config option definitions uppercase.

File: packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/src/debug.c [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#include "x86_64_accton_as5912_54xk_int.h"
```
Comment (jnealtowns) [inline packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/src/debug.c]: Please remove this file. It doesn't do anything and the debug option has been moved to the SYSI interface.

File: packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/src/platform_lib.c [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+ ***********************************************************/
+#include <unistd.h>
+#include <fcntl.h>
+#include <onlplib/file.h>
+#include <onlp/onlp.h>
+#include "platform_lib.h"
+
+#define PSU_NODE_MAX_PATH_LEN 64
+
+int _onlp_file_write(char *filename, char *buffer, int buf_size, int data_len)
```
Comment (jnealtowns) [inline packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/src/platform_lib.c]: Please use the existing onlp_file_* APIs for this functionality.

File: packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/src/sfpi.c [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+{
+    /*
+     * Return 1 if present.
+     * Return 0 if not present.
+     * Return < 0 if error.
+     */
+    int present;
+    char *path = sfp_get_port_path(port, "sfp_is_present");
+
+    if (onlp_file_read_int(&present, path) < 0) {
```
Comment (jnealtowns) [inline packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/src/sfpi.c]: The ONLP File APIs allow you to specify a format string directly. You can replace the construction of "path" here with a direct call.

**Comment by sonoble** [comment]
Hi @brandonchuang , can you update this pull?  I have confirmed it working in the lab, but we need it to pass our coding standards.

Thank you

**Comment by brandonchuang** [comment]
Hi @sonoble ,
I had modified the source code. Could you help verify again?
Thank you

**Comment by jnealtowns** [comment]
@brandonchuang Your recent commit removes debug.c but does not address the other review items.

**Comment by jnealtowns** [comment]
ping @brandonchuang

**Comment by akenliu** [comment]
Hi Jnealtowns,
Sorry for the inconvenience.
Brandon is out of the office until Monday 9/11.

**Comment by jnealtowns** [comment]
@akenliu  No problem :-)

**Comment by brandonchuang** [comment]
Hi @jnealtowns , 
I'm so sorry I forgot to commit all the remaining files :(
Could you check it again? Thanks.

**Comment by jnealtowns** [comment]
Thank you @brandonchuang

**Pull request merged**